tests/scripts/depends.py

```diff
@@ -630,8 +630,11 @@ def __init__(self, options, conf):
             # across various modules, but it depends on either SHA256 or SHA512.
             # As a consequence an "exclusive" test of anything other than SHA256
             # or SHA512 with MBEDTLS_ENTROPY_C enabled is not possible.
+            # Note for update: when MBEDTLS_SHA3_C is removed the mutual_exclusion
```

Comment:
We need a common operator to indicate things that need to be done in the documentation. Until that is decided shoud we use `TODO:` which is global?